src/material/sidenav/drawer.ts

```diff
@@ -448,13 +463,20 @@ export class MatDrawer implements AfterContentInit, AfterContentChecked, OnDestr
 
   /** Whether focus is currently within the drawer. */
   private _isFocusWithinDrawer(): boolean {
-    const activeEl = this._doc?.activeElement;
+    const activeEl = this._doc.activeElement;
```

Comment:
@crisbeto Is this change save? `_doc` still has the @Optional decorator and could in theory be 'null'. At other code locations checks are still present.